androidApp/src/main/java/com/mbta/tid/mbta_app/android/more/MoreSectionView.kt

```diff
@@ -38,7 +38,7 @@ fun MoreSectionView(section: MoreSection, toggleSetting: ((Settings) -> Unit)) {
 
     val note = section.note
 
-    if (!(section.requiresStaging && appVariant != AppVariant.Staging)) {
```

Comment:
suggestion (non-blocking): Might be worth renaming `requiresStaging` at this point too. We could invert to `requiresProd`